lxd/main_args.go

```diff
@@ -133,4 +133,6 @@ Internal commands (don't call these directly):
         Indicate that a migration dump was successful
     netcat
         Mirror a unix socket to stdin/stdout
+    proxy_dev_start
+        Start proxy device process for a container
```

Comment:
I'd call it `forkproxy` instead, for consistency with all the other similar container related subcommands we already have.